pkg/model/iam/tests/iam_builder_node_strict.json

```diff
@@ -0,0 +1,55 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeInstances"
```

Comment:
DescribeRegion?  Why do we need DescribeInstances?

Reply:
This was introduced in #2497 with a comment that Protokube makes a DescribeInstances call. If it's not needed at all we can drop the Node EC2 policy completely.

Reply:
Please check. We may only need for gossip.

Reply:
This permission is still required, I got the below error when dropping it for compute nodes:
`kubelet[8706]: error: failed to run Kubelet: could not init cloud provider "aws": error finding instance i-12345678: error listing AWS instances: UnauthorizedOperation: You are not authorized to perform this operation`